modules/oe_theme_content_event/src/Plugin/ExtraField/Display/DetailsExtraField.php

```diff
@@ -46,10 +52,13 @@ class DetailsExtraField extends EventExtraFieldBase implements ContainerFactoryP
    *   The entity type manager.
    * @param \Drupal\Core\Entity\EntityRepositoryInterface $entity_repository
    *   The entity repository service.
+   * @param \Drupal\Core\Render\RendererInterface $renderer
+   *   The renderer service.
    */
-  public function __construct(array $configuration, $plugin_id, $plugin_definition, EntityTypeManagerInterface $entity_type_manager, EntityRepositoryInterface $entity_repository) {
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, EntityTypeManagerInterface $entity_type_manager, EntityRepositoryInterface $entity_repository, RendererInterface $renderer) {
```

Comment:
Like I said, you need to default the new argument to NULL and check if it doesnt not implement the correct interface, load it statically.

Reply:
Fixed.